README.md

```diff
@@ -1,62 +1,111 @@
-![dyrectorio logo](dyrectorio.svg)
+<p align="center">
+  <a href="http://docs.dyrector.io/" target="blank">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="dyrectorio-dark.png">
+      <source media="(prefers-color-scheme: light)" srcset="dyrectorio-light.png">
+      <img alt="dyrector.io official logo" src="dyrectorio-dark.png" width="400">
+    </picture>
+  </a>
+</p>
+
+<p align="center">Open-source CD platform for developers and non-specialists to manage multi-instance deployments, microservices and configurations in Kubernetes, Docker.</p>
+
+<p align="center">
+  <a href="https://github.com/dyrector-io/dyrectorio/tags" target="_blank"><img src="https://img.shields.io/github/v/tag/dyrector-io/dyrectorio" alt="GitHub tag (latest by date)"/></a>
+    <a href="./LICENSE" target="_blank"><img src="https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)" alt="License"/></a>
+    <a href="https://discord.gg/pZWbd4fxga" target="_blank"><img src="https://img.shields.io/discord/797082431902449694" alt="Discord Budget"/></a>
+    <a href="https://codecov.io/gh/dyrector-io/dyrectorio" target="_blank"><img src="https://img.shields.io/codecov/c/github/dyrector-io/dyrectorio/develop?logo=Go&token=F5TZTAJTKX" alt="codecov"/></a>
+    <a href="https://github.com/dyrector-io/dyrectorio/graphs/contributors" target="_blank"><img src="https://img.shields.io/github/contributors/dyrector-io/dyrectorio" alt="GitHub contributors"/></a>
+    <a href="https://github.com/dyrector-io/dyrectorio/issues" target="_blank"><img src="https://img.shields.io/github/issues-raw/dyrector-io/dyrectorio" alt="GitHub issues"/></a>
+    <a href="https://twitter.com/dyrectorio" target="_blank"><img src="https://img.shields.io/twitter/follow/dyrectorio?style=social" alt="Twitter Follow"/></a>
+</p>
 
-# dyrector.io - The open source internal delivery platform
+## Overview
 
-[![License](https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)](LICENSE)
-[![Discord Budget](https://img.shields.io/discord/797082431902449694)](https://discord.gg/pZWbd4fxga)
-[![codecov](https://img.shields.io/codecov/c/github/dyrector-io/dyrectorio/develop?logo=Go&token=F5TZTAJTKX)](https://codecov.io/gh/dyrector-io/dyrectorio)
+Life isn’t about application management. Why should your team focus so much of its precious time and effort on it when things can be simpler?
 
-## Overview
+dyrector.io is for you if you’d like to shift your team’s focus from repeatable steps of multi-instance deployments to delivering value to your users. And when a deployment is successful or any error occurs, the team gets notified on chat so they can react as soon as possible.
```

Comment:
I know there is no grammatical rule for starting a sentence with an "and" but can you please change it to "also" or something similiar . Its kinda weird to my eyes

Reply:
Good point I'll fix it. Thanks!

Reply:
Updated could you please check it?